MM/5.8.0/config/media_manager_5/fields/asset_list_fields.dcl

```diff
@@ -0,0 +1,10 @@
+resource configservice_string_config_field asset_list_fields {
+    default_value = '[]'
+    type = 'String'
```

Comment:
Perhaps this should be of type Metafield?

see

https://github.com/Digizuite/configurations/blob/master/MM/5.8.0/config/media_manager_5/fields/asset_info_meta_fields.dcl

Reply:
@madjufra I need to store objects of custom type, not metafields - therefore I store it as JSON - String

Reply:
All right! Good explanation :)